ql/src/java/org/apache/hadoop/hive/ql/queryhistory/repository/QueryHistoryRepository.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package org.apache.hadoop.hive.ql.queryhistory.repository;
+
+import java.util.Queue;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.ql.queryhistory.schema.QueryHistoryRecord;
+import org.apache.hadoop.hive.ql.queryhistory.schema.QueryHistorySchema;
+
+public interface QueryHistoryRepository {
+  String QUERY_HISTORY_DB_NAME = "sys";
+  String QUERY_HISTORY_TABLE_NAME = "query_history";
+  String QUERY_HISTORY_DB_TABLE_NAME =
+      String.format("%s.%s", QUERY_HISTORY_DB_NAME, QUERY_HISTORY_TABLE_NAME);
+
+  String QUERY_HISTORY_DB_COMMENT = "Hive SYS database";
```

Comment:
do we need this or could just use null

Reply:
I think a non-null message is better than a null if it holds at least a minimal information
here, given a shared HMS, this message at least tells it's a "Hive" thing
this question implies another one I was thinking about while implementing this service, which is: whether we should refactor sys database things to a dedicate place (where we can have this constant for example)
finally I decided to not cope with this in the scope of this ticket

Reply:
okay, FYI:
```
0: jdbc:hive2://localhost:10000/default> describe database sys;
...
+----------+--------------------+-------------------+------------------+--------------+-------------+-----------------+----------------+
| db_name  |      comment       |     location      | managedlocation  |  owner_name  | owner_type  | connector_name  | remote_dbname  |
+----------+--------------------+-------------------+------------------+--------------+-------------+-----------------+----------------+
| sys      | Hive SYS database  | location/in/test  |                  | laszlobodor  | USER        |                 |                |
+----------+--------------------+-------------------+------------------+--------------+-------------+-----------------+----------------+
```

Reply:
sys db is created via schema scripts, it should already exist